app/assets/javascripts/discourse-plugins/index.js

```diff
@@ -62,6 +62,22 @@ function namespaceModules(tree, pluginDirectoryName) {
   });
 }
 
+function parsePluginName(pluginRbPath) {
+  const pluginRb = fs.readFileSync(pluginRbPath, "utf8");
+  // Match parsing logic in `lib/plugin/metadata.rb`
+  for (const line of pluginRb.split("\n")) {
+    if (line.startsWith("#")) {
```

Comment:
I wish we could go back in time and move plugin metedata out of `plugin.rb` and into a separate machine-parsable file 😅